CONTRIBUTING.md

```diff
@@ -55,6 +55,12 @@ To contribute a change to the project, *please follow these steps*:
 
   7. When ready, create your pull request and request a review from "rt-contributors".
 
+Note the code philosophy outlined in the first section of the first book. In short, the code is
+intended to be clear for everyone, using as little C/C++ arcania as possible. We have chosen to
```

Comment:
[sp]? arcana?

Esoterica? C/C++isms ?

Reply:
Changed to "using simple C/C++ idioms as much as possible."